workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -725,16 +761,16 @@ def register_parser(subparsers):
 
 def main(args):
     computed_diagnostics = ComputedDiagnosticsList.from_directory(args.input)
-    make_report(computed_diagnostics, args.output)
+    make_report(computed_diagnostics, args.output, args.gsrm)
```

Comment:
Overall this looks good, but I am wondering if it would be preferable to avoid adding an API-level argument (`args.gsrm`), in favor of trying to infer the needed grid variables from the data at the lower level where it's actually used, for example in `plot_cubed_sphere_maps` by wrapping the `ds = run_diags.get_variables(run, COORD_VARS + [varname])`  with a try-except or something.

Reply:
Good idea, I think we can actually just check if `ncol` is in the dataset to switch to scream specific plotting. Will make the change.